db/dev_seeds/settings.rb

```diff
@@ -16,7 +16,9 @@
     "twitter_handle": "@consuldemocracy_dev",
     "twitter_hashtag": "#consuldemocracy_dev",
     "votes_for_proposal_success": "100",
-    "youtube_handle": "CONSUL DEMOCRACY"
+    "youtube_handle": "CONSUL DEMOCRACY",
+    "map.longitude": "-3.703790",
+    "map.latitude": "40.416775"
```

Comment:
What do you think about changing the seeds in `db/dev_seeds/budgets.rb`, so they use `Setting["map.latitude"]` and `Setting["map.longitude"]`? Then we wouldn't have to overwrite the default settings during development :thinking:, and it would be one less reference to Madrid :joy:.